### DISCUSSION TITLE
I tried to summon the jade_ore in world_the_end but every time I go to end city and leave the dragon area the server crashes. I think it's because all the end stones are replaced with jade_ore but I'm not sure

### DISCUSSION THREAD

**Discussion by ofek111** [DerFrZocker/Custom-Ore-Generator#29] [Q&A]
# Only edit this file if you 10000% sure you know what you are doing. 
# You can set everything via the plugin commands, no need to edit this file manully.

# Only edit this file if you 10000% sure you know what you are doing. 
# You can set everything via the plugin commands, no need to edit this file manully.
value:
  ==: CustomOreGenerator#OreConfig
  name: jade_ore
  material: NOTE_BLOCK
  ore-generator: SINGLE_ORE_GENERATOR
  block-selector: COUNT_RANGE
  activated: true
  generated-all: false
  biomes:
  - END_HIGHLANDS
  - END_MIDLANDS
  - END_BARRENS
  - SMALL_END_ISLANDS
  replace-materials:
  - END_STONE
  custom-data:
    ORAXEN: jade_ore
  ore-generator-settings:
    ==: CustomOreGenerator#OreSettingsContainer
    ore-settings:
      VEIN_SIZE: 20.0
  block-selector-settings:
    ==: CustomOreGenerator#OreSettingsContainer
    ore-settings:
      VEINS_PER_CHUNK: 1.0
      HEIGHT_RANGE: 50.0
      MINIMUM_HEIGHT: -50.0

**Comment by DerFrZocker** [comment]
Your minimum height should be positive since the end does not go below 0, currently you generated ore between -50 and 0, where no end stone is.

Other than that, I cannot help you further with out the crash / error log.
